src/__swaps__/utils/numbers.ts

```diff
@@ -285,6 +285,27 @@ export const convertAmountToNativeDisplay = (
   return `${display} ${nativeSelected.symbol}`;
 };
 
+/**
+ * @desc convert from amount value to display formatted string
+ */
+export const convertAmountToNativeDisplayWorklet = (value: number, nativeCurrency: keyof nativeCurrencyType) => {
```

Comment:
to keep these types of functions consistent, let's have this support `number | string` and handle that inside of this function so we don't have to do extra work outside of this function every time